platforms/corda-ent/configuration/roles/helm_component/tasks/main.yaml

```diff
@@ -20,6 +20,8 @@
   template:
     src: "{{ helm_templates[type] | default('helm_component.tpl') }}"
     dest: "{{ values_dir }}/{{ name }}/{{ component_name }}.yaml"
+  vars:
+    docker_image: "{{ corda_enterprise_image[corda_service_version] }}"
```

Comment:
where is the var for for this?

Reply:
i think the file was not checked in, missed saving the file I guess so didn't got picked my git add

Reply:
its in the vars folder of helm_component only